README.md

```diff
@@ -1,114 +1,142 @@
-_Note: this repo uses submodules, clone it with `--recursive` or do a `git submodule update --init` after cloning._
+- [Audio Visualizer](https://mathbox.org/demo/audio-visualizer.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/demo/audio-visualizer.html))
+- [Cylindrical Stream](https://mathbox.org/demo/cylindrical-stream.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/demo/cylindrical-stream.html))
+- [Data/Shape Mapping](https://mathbox.org/demo/shapes.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/demo/shapes.html))
+- [LaTeX/HTML/GL Labels](https://mathbox.org/demo/labels.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/demo/labels.html))
+- [Quaternion Hypersphere](https://mathbox.org/math/quat.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/math/quat.html))
+- [Render-to-Texture History](test/resample2.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/test/resample2.html))
+- [Vertex Warping](test/vertex.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/test/vertex.html))
+- [Volumetric Vectors](test/volume.html) ([code](https://github.com/unconed/mathbox/blob/master/examples/test/volume.html))
 
----
+And many more at https://mathbox.org.
 
 ## Installation
 
-- via npm:
-
-  ```bash
-  npm install mathbox
-  ```
-
-- or CDN:
-  ```html
-  <script
-    type="text/javascript"
-    src="https://cdn.jsdelivr.net/npm/mathbox@latest/build/bundle/mathbox.js"
-  ></script>
-  <link
-    rel="stylesheet"
-    href="https://cdn.jsdelivr.net/npm/mathbox@latest/build/mathbox.css"
-  />
-  ```
+You can install MathBox via [npm][npm-url] for use with a bundler like
+[Webpack](https://webpack.js.org/), or include a global `MathBox` object onto
+your page by including the library via CDN.
 
-Open the included `/examples` to see more demos.
+### NPM Package
 
----
+- Run the following in your project's directory to install MathBox and
+  [Three.js][three-url] via [npm][npm-url]:
 
-## Docs & Help
+```bash
+npm install --save mathbox three
+```
 
-See:
+Import `THREE` and `MathBox` (library and stylesheet), along with a controls
+instance that you'll pass to the `MathBox.mathBox` constructor:
 
-- [Quick Start](docs/intro.md) for a hands on introduction.
-- [Glossary](docs/glossary.md) of terms to help get familiar with MathBox and WebGL.
-- [MathBox API](docs/api.md) for typical usage.
-- [List of Primitives](docs/primitives.md) for a full element reference.
-- [Writing Custom Shaders](docs/shaders.md) for info on custom shaders and GPU-side processing.
-- [Context API](docs/context.md) for advanced usage.
+```js
+import "mathbox/mathbox.css"
 
-Join us in the [MathBox Google Group](https://groups.google.com/forum/#!forum/mathbox) or #mathbox on Freenode.
+import { * as THREE } from "three"
+import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js"
+import { * as MathBox } from "mathbox"
```

Comment:
**Requested Change:**

```suggestion
import * as THREE from "three"
import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js"
import * as MathBox from "mathbox"
```

(`import { * as THREE } from "three"` is invalid js)

Reply:
Whoops, clearly I was going from memory here :)